cli/known_plugins.py

```diff
@@ -0,0 +1,6 @@
+plugin_info = {
```

Comment:
this file is still needed?

Reply:
I could put this dict in the `__init__` file, but I thought for the people that want to add their plugin it is nicer if it is in this separate file.

Reply:
ok, but is not extracted from github right?

Reply:
No, imported and used locally: `from .known_plugins import plugin_info`